src/tox_lsr/test_scripts/runcontainer.sh

```diff
@@ -188,6 +193,15 @@ while [ -n "${1:-}" ]; do
             CONFIG="$1" ;;
         --erase-old-snapshot)
             ERASE_OLD_SNAPSHOT=true ;;
+        --containers)
+            shift
+            CONTAINER_COUNT="$1" ;;
+        --exclude)
+            shift
+            EXCLUDES+=("$1") ;;
```

Comment:
I think it is ok to leave this, but in general, the test framework will use `--skip-tags ...` to exclude tests that cannot be run in containers.